server/src/main/java/org/elasticsearch/reservedstate/TransformState.java

```diff
@@ -24,8 +22,4 @@
  * other state outside the cluster state. The consumer, if provided, must return a {@link NonStateTransformResult} with
```

Comment:
Reference to NonStateTransformResult here

Reply:
Removed